docs/Daos.md

```diff
@@ -0,0 +1,252 @@
+# Working with DAOs
+
+Arc.js provides a class called [DAO](api/classes/DAO) that facilitates creating and working with DAOs.  Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.
```

Comment:
> api/classes/DAO

  is there such a link ?

Reply:
> Read on for more information about DAOs in DAOstack and the [DAO](api/classes/DAO) class.

I think this sentence can be removed..

Reply:
> is there such a link ?

Yes

> I think this sentence can be removed..as you already gave a link and it is not needed.

I use these transition sentences all the time.  I was taught in school that this is an important part of creating flow in technical writing.

Reply:
Please point me to this link in the branch .

Reply:
this is not valid on this repo (GitHub.com)

Reply:
As requested, I added a readme file to the docs folder.  You can see it here:  https://github.com/daostack/arc.js/tree/documentation/docs